package.json

```diff
@@ -32,7 +32,7 @@
     "test-unit": "nyc --nycrc-path=.nycrc.js node npm/test-unit.js"
   },
   "dependencies": {
-    "faker": "5.5.3",
+    "@faker-js/faker": "^5.5.3",
```

Comment:
Can you please pin the version and update the package-lock as well?

```suggestion
    "@faker-js/faker": "5.5.3",
```

Reply:
Sure.

Reply:
Done.